server/NetPacksLobbyServer.cpp

```diff
@@ -115,6 +115,9 @@ bool LobbyChatMessage::checkClientPermissions(CVCMIServer * srv) const
 
 bool LobbySetMap::applyOnServer(CVCMIServer * srv)
 {
+	if (srv->state != EServerState::LOBBY) {
+		return false;
+	}
```

Comment:
Should be formatted according to our guidelines.

Reply:
Done